src/backend/mailbox_manager/utils/dimail.py

```diff
@@ -210,7 +210,7 @@ def send_new_mailbox_notification(self, recipient, mailbox_data):
                 exception,
             )
 
-    def synchronize_mailboxes_from_dimail(self, domain):
+    def import_mailboxes(self, domain):
         """Synchronize mailboxes from dimail - open xchange to our database.
```

Comment:
@sdemagny Nice renaming, please update the comment also :)

Reply:
Done